gtsfm/densify/mvs_base.py

```diff
@@ -0,0 +1,62 @@
+"""Base class for the MVS stage of the back-end.
+
+Authors: John Lambert, Ren Liu
+"""
+import abc
+from typing import Dict, Tuple
+
+import dask
+import numpy as np
+from dask.delayed import Delayed
+from gtsam import PinholeCameraCal3Bundler
+
+from gtsfm.common.image import Image
+
+
+class MvsBase(metaclass=abc.ABCMeta):
+    """Base class for all multi-view stereo hyperparameters."""
```

Comment:
Is it a base class for hyperparams or for all MVS modules?

Reply:
Good point, should be "Base class for all multi-view stereo implementations."